src/test/java/io/strimzi/kafka/bridge/http/HttpCorsTests.java

```diff
@@ -46,6 +48,7 @@
     static {
         config.put(HttpConfig.HTTP_CONSUMER_TIMEOUT, timeout);
         config.put(BridgeConfig.BRIDGE_ID, "my-bridge");
+        config.put(KafkaConfig.KAFKA_CONFIG_PREFIX + ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9092");
```

Comment:
why this addition? It doesn't seem to be related to admin stuff.

Reply:
It is related, as adminClient is now initialized when HttpBridge is started. Unlike the other HttpBridge tests, HttpCorsTests doesn't use HttpBrdigeTestBase which sets the bootstrap address, so this bootstrap address is set here.